app/labor/badge_rewarder.rb

```diff
@@ -1,12 +1,13 @@
 module BadgeRewarder
-  YEARS = { 1 => "one", 2 => "two", 3 => "three", 4 => "four", 5 => "five", 6 => "six", 7 => "seven", 8 => "eight", 9 => "nine", 10 => "ten" }.freeze
+  YEARS = { 1 => "one", 2 => "two", 3 => "three" }.freeze
```

Comment:
I think @benhalpern mentioned yesterday that the 4 year badge will be coming up soon (https://dev-internal.slack.com/archives/C69MABC03/p1591122156079600). Not sure if we want to amend it at this point in preparation for 4 years or just open up a new PR later but thought I'd mention it anyways :)

Reply:
Whoever adds the 4 year badge will already have to make the PR, since the awaring is currently hard-coded for 1 - 3 years (see above), so I'd say we can add it then.